Main.py

```diff
@@ -0,0 +1,94 @@
+import pygame, sys, time, math, random
+from pygame.locals import *
+
+pygame.init()#Initiates Pygame
+img1 = pygame.image.load('BigSmoke.jpg')  #Imports BigSmoke
```

Comment:
If you numbered the file names (e.g. 0.jpg, 1.jpg, etc.), you could easily make these loading operations a loop.